### ISSUE TITLE
ctx.lineTo does not remember previous position

### ISSUE THREAD

**Issue by carlreinecken** [davidenke/context-filter-polyfill#28]
When I try to draw on my canvas, where each `touchmove` event calls `ctx.lineTo` and `ctx.stroke`, only a stroke is drawn for the first point. I assume this is because the context of where the path is, is lost after the drawing function `stroke` is called? But maybe I missing something else?

I was able to get the drawing working again by calling `ctx.moveTo` and manually setting to the previous point before each `ctx.lineTo` call. However, for other reasons this workaround didn't solve this issue for me.

After looking at your code a bit, I set `Object.defineProperty(canvasElement, '__skipFilterPatch', { value: true })` on my drawing canvas because I don't need the `ctx.filter` on this particular canvas at all, I just use it with another canvas. This solves it for me at least. On the other canvas your polyfill works really well, thank you very much for this library!

**Comment by carlreinecken** [comment]
I was able to get the drawing working again by calling `ctx.moveTo` and manually setting to the previous point before each `ctx.lineTo` call. However, the performance for my undo function.... doesn't matter!

After looking at your code a bit, I set `Object.defineProperty(canvasElement, '__skipFilterPatch', { value: true })` on my drawing canvas because I don't need the `ctx.filter` on this particular canvas at all, I just use it with another canvas. This solves it for me at least. On the other canvas your polyfill works really well, thank you very much for this library!
